TestAssets/NuGet.Config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same here, not sure if this is correct. That means that when we set the PackageSources from the build from sources, the TestAssets are not going to respect it.